eating_cookies/eating_cookies.py

```diff
@@ -2,10 +2,24 @@
 Input: an integer
 Returns: an integer
 '''
+
+from itertools import combinations
+
 def eating_cookies(n):
-    # Your code here
+    # * base case(s)
+    if n <= 1:
+        return 1
+    if n == 2:
+        return 2
+
+
+    # * create a list with a length of n
+    arr = list(range(0, n-1))
+
+    # * use itertools.combinations to create a list of 2-length tuple subsets
+    tup_arr = list(combinations(arr, 2))
 
-    pass
+    return len(tup_arr)
```

Comment:
Absolutely _killer_ use of the `combinations` function from `itertools`.

In an interview scenario, the only thing to look out for is that they might actually want you to implement it from scratch. 

It's not imperative, but you might want to take some time and see if you can get do it without `itertools.combinations`